releases/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,205 @@
+
+```tsx
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
+import LocalExtension from "@itwin/test-local-extension";
+import { Viewer } from "@itwin/web-viewer-react";
+import React, { useCallback, useMemo, useState } from "react";
+
+export const App: React.FC = () => {
+  const [iModelId, setIModelId] = useState(
+    process.env.IMJS_AUTH_CLIENT_IMODEL_ID
+  );
+  const [iTwinId, setITwinId] = useState(process.env.IMJS_AUTH_CLIENT_ITWIN_ID);
+
+  const authClient = useMemo(
+    () =>
+      new BrowserAuthorizationClient({
+        scope: process.env.IMJS_AUTH_CLIENT_SCOPES ?? "",
+        clientId: process.env.IMJS_AUTH_CLIENT_CLIENT_ID ?? "",
+        redirectUri: process.env.IMJS_AUTH_CLIENT_REDIRECT_URI ?? "",
+        postSignoutRedirectUri: process.env.IMJS_AUTH_CLIENT_LOGOUT_URI,
+        responseType: "code",
+        authority: process.env.IMJS_AUTH_AUTHORITY,
+      }),
+    []
+  );
+
+  return (
+    <div style={{ height: "100vh" }}>
+      <Viewer
+        authClient={authClient}
+        iTwinId={iTwinId ?? ""}
+        iModelId={iModelId ?? ""}
+        enablePerformanceMonitors={true}
+        extensions={[
+          new LocalExtensionProvider({
+            manifestPromise: LocalExtension.manifestPromise,
+            main: LocalExtension.main,
+          }),
+          new RemoteExtensionProvider({
+            jsUrl: "http://localhost:3001/dist/index.js",
+            manifestUrl: "http://localhost:3001/package.json",
+          }),
+        ]}
+      />
+    </div>
+  );
+};
+```
+
+### Loading Bar
+
+- We've changed the style of the default loading bar that displays when an iModel is loading
+
+### renderSys configuration
+
+- Added support for iTwin.js renderSys configuration
+
+### New error components
+
+- Replaced the error handling components from the @itwin/error-handling-react package with similar components from the iTwin UI package
+
+## How to upgrade an exisiting application
+
+- Change the version of `@itwin/web-viewer-react` or `@itwin/desktop-viewer-react` in your package.json to the following:
+
+  ```
+    "dependencies": {
+      ...
+      "@itwin/web-viewer-react": "^3.0.0",
+      ...
+    },
+  ```
+
+  ```
+  "dependencies": {
+    ...
+    "@itwin/desktop-viewer-react": "^3.0.0",
+    ...
+  },
+  ```
+
+- React to the [breaking changes](#breaking-changes) above.
+
+## Quick start a new application
+
+- iTwin Viewer for Web
+
+  ```sh
+  npx create-react-app web-viewer-app --template @itwin/web-viewer --scripts-version @bentley/react-scripts
+  ```
+
+- iTwin Viewer for Desktop
+
+  ```sh
+  npx create-react-app desktop-viewer-app --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts
+  ```
+
+## Bug Fixes
+
+- Fixed an issue where backstage item labels were not showing
+- No longer using a seed view by default when generating the default view of an iModel
+
+## Optimizations
+
+- Consolidated the types for the Viewer and the BlankViewer
```

Comment:
not just the types, components too. wonder if this needs a bigger shout out.
web support connected and blank connections
desktop supports connected, blank, and local. i think desktop supporting blank connections is new

Reply:
I added a section the "removed" section about the BlankViewer. Let me know if its sufficient